takeshape/penny
# reorg nav and search, plus storybooks

## Test Plan

1. Pull this branch
2. Review the stories and new organization
3. Make sure the homepage and search modal aren't broken
